documentation/modules/con-console.adoc

```diff
@@ -0,0 +1,12 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-using-console.adoc
+
+[id='con-console-{context}']
+= {ConsoleName} user permissions
+
+{ConsoleName} uses the OpenShift RBAC permissions model. For more information about the OpenShift RBAC permissions model, see the link:https://docs.openshift.com/container-platform/3.11/admin_guide/manage_rbac.html[OpenShift documentation^].
+
+To access the {ConsoleName}, the OpenShift user requires `addressspace` and `address` roles. The specific permissions given to the role govern the extent of the access. For example, for edit access, `edit` permissions need be to given to the role associated with the user, and for view-only access, `list` permissions need to be granted. For more information about the provided example roles, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-example-roles-messaging[{ProductName} example roles].
```

Comment:
"the OpenShift user requires `addressspace` and `address` "  .. In reality the user requires addresspace permission to reach the first page in the console (that lists the addressspaces_, then they need address permission to click through see the addresses, but I think what you have said is good enough detail for here.

Reply:
Ok, thanks for the clarification. I left it as is.